site/en/adminGuide/tls.md

```diff
@@ -0,0 +1,469 @@
+---
+id: tls.md
+title: Enable TLS proxy
+summary: Learn how to enable TLS proxy in Milvus.
+---
+
+# Enable TLS proxy
+
+TLS (Transport Layer Security) is a type of mutual authentication in which two parties in a connection verify each other using the TLS protocol. Milvus proxy uses the TLS mutual authentication.
+
+This topic describes how to enable TLS proxy in Milvus.
+
+## Create your own certificate
+
+### Prerequisites
+
+Make sure OpenSSL is installed. If you have not installed it, run the following command first.
+
+```
+sudo apt install openssl
```

Comment:
explain this. command is an example and it runs on Ubuntu

Reply:
Can I just delete this example and put a link to https://github.com/openssl/openssl/blob/master/INSTALL.md?

Reply:
Sure